agent_server.py

```
@@ -1212,13 +1242,16 @@ async def _run_user_plugin_dispatch():
                         else:
                             logger.warning(f"[TaskExecutor] ❌ UserPlugin failed: {up_result.error}")
                             try:
+                                _fail_summary = f'插件任务 "{plugin_id}" 执行失败'
+                                if detail:
+                                    _fail_summary = f'插件任务 "{plugin_id}" 执行失败：{detail}'
```

Ruff (0.15.5)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)